README.md

```diff
@@ -18,12 +18,12 @@ npm test
 # Usage
 
 ```javascript
-import mathtofhirpath
-mathtofhirpath.fhirconvert("(a+b)^2", [a, b]); // returns ($a+$b).power(2)
-mathtofhirpath.fhirconvert("CEILING(a*b)", [a, b]); // returns ($a*$b).ceiling()
+import * as easyPathExpressions from 'easy-path-expressions';
+easyPathExpressions.fhirconvert("(a+b)^2", [a, b]); // returns ($a+$b).power(2)
```

Comment:
I was testing this earlier.  [a, b] should be ["a", "b"].

Reply:
Good catch, updated.